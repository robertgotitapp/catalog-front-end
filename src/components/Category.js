import React, { Component } from 'react';

class Category extends Component {
  render() {
    return (
      <div>
        This is Category
      </div>
    );
  }
}

export default Category;
