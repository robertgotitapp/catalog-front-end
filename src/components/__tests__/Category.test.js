import React from 'react';
import { shallow } from 'enzyme';
import { Category } from '../Category';

describe('components/Category', () => {
  const props = {
    category: {
      created: '2019-06-12T03:17:40+00:00',
      description: 'fsfdfdfasdf',
      id: 1,
      name: 'Umbrella',
      updated: '2019-06-12T03:17:40+00:00',
    },
  };
  const container = shallow(<Category {...props} />);

  it('should render correctly', () => {
    expect(container).toMatchSnapshot();
  });

  it('should contain category name', () => {
    expect(container.find('div')).toHaveLength(1);
  });
});
