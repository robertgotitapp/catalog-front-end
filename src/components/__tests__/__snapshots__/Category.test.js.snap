// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Category should render correctly 1`] = `
<div
  onClick={[Function]}
>
  Umbrella
</div>
`;
